staramr/results/AMRDetectionSummary.py

```diff
@@ -130,6 +130,7 @@ def create_detailed_summary(self, include_negatives: bool=True) -> DataFrame:
         else:
             resistance_frame = self._resfinder_dataframe.copy()
             resistance_frame['Data Type']='Resistance'
+            resistance_frame = resistance_frame.round({'%Identity': 2, '%Overlap': 2})
```

Comment:
Could you maybe move the decimal places to round to into a constant in this class? E.g., 

```python
class AMRDetectionSummary:
    SEPARATOR = ','
    FLOAT_DECIMALS = 2

# ...

resistance_frame.round({'%Identity': self.FLOAT_DECIMALS, '%Overlap': self.FLOAT_DECIMALS})
```

Reply:
Yeah fixed it in commit 3e1ae93